src/components/todo/list.js

```diff
@@ -1,34 +1,34 @@
-import React, {useState} from 'react';
-import {Row, Col, Toast, Button, Badge} from 'react-bootstrap'
+import React, { useState } from 'react';
+import { Row, Col, Toast, Button, Badge } from 'react-bootstrap'
 
 
-function TodoList (props) {
+function TodoList(props) {
 
-  
-  const [showA, setShowA] = useState(true);
-  const toggleShowA = () => setShowA(!showA);
+  console.log(props.list)
+  // Need to update so that Badges can be changed as well as so that hitting x on an item removes it from the Database instead of hiding all items.
 
 
-    return (
+  return (
     <Row>
-        {props.list.map(item => (
-          <Toast show={showA} onClose={toggleShowA}>
-          <Toast.Header>
+      {props.list.map(item => (
+        <Toast key={item._id}>
+          <Toast.Header closeButton= {false}>
             <img
               src="holder.js/20x20?text=%20"
               className="rounded mr-2"
               alt=""
             />
-            
-            <Badge pill variant={item.variant} className="mr-3">{item.status}</Badge>
+
+            <Badge pill variant={item.complete ? "danger": "success" } className="mr-3">{item.complete ? "Completed" : "Pending"}</Badge>
             <strong className="mr-auto">{item.assignee}</strong>
-            
+            <button onClick={() => props.handleDelete(item._id)} type="button" class="close ml-2 mb-1" data-dismiss="toast"><span aria-hidden="true">×</span><span class="sr-only">Close</span></button>
           </Toast.Header>
+
           <Toast.Body onClick={() => props.handleComplete(item._id)}>{item.text}</Toast.Body>
```

Comment:
Move the `onClick` part to line 22 inside <Badge> this will help toggle between 'pending' and 'completed'